docs/installation.md

```diff
@@ -3,10 +3,32 @@
 **NB** - The package is currently undergoing heavy revision and these installation instructions apply to installing and
 workin on the the `dev` branch.
 
-Currently TopoStats should be installed from the git repository, ideally under a [Python Virtual
-Environment](https://realpython.com/python-virtual-environments-a-primer/)
-(e.g. [Miniconda](https://docs.conda.io/en/latest/miniconda.html)).
+TopoStats is a [Python](https://www.python.org) package designed to run at the command line. You may have Python
+installed on your system but should use a [Python Virtual
+Environment](https://realpython.com/python-virtual-environments-a-primer/) such as
+[Miniconda](https://docs.conda.io/en/latest/miniconda.html) and install it under the Virtual Environment. The versions
+of Python supported is Python >=3.8 and so when creating your virtual environment you should specify this `3.8` as the
+minimum.
 
+## Setting up Conda
+
+Once you have downloaded and installed [Miniconda](https://docs.conda.io/en/latest/miniconda.html) you can create a
+virtual environment for installing TopoStats with a version of Python that meets the requirements with. We will call
+this environment `topostats` (specified with the `--name topostats` option) and to use Python 3.10 (the option
+`python=3.10`). After creating it we can, as per the instructions printed out, activate the environment.
+
+``` bash
+conda create --name topostats python=3.10
+conda activate topostats
+```
+
+You are now ready to clone and install TopoStats.
+
+## Cloning and Installing
```

Comment:
This is an option that doesn't need `git`:

```
pip install git+https://github.com/AFM-SPM/topostats@dev
```

However, on reflection maybe not the best one as it loses the version hash which we might need to help fix user problems (until we switch to semantic versioning, which is captured by `pip`).